platforms/platforms.go

```diff
@@ -3,6 +3,7 @@ package platforms
 
 import (
 	"fmt"
+	sem_ver "github.com/hashicorp/go-version"
```

Comment:
please remove the underscore (our internal Go linter will complain otherwise).

Reply:
fixed